app/assets/javascripts/host_edit.js

```diff
@@ -442,7 +442,7 @@ function reload_host_params(){
 function reload_puppetclass_params(){
   var host_id = $("form").data('id');
   var url2 = $('#params-tab').data('url2');
-  var data = $("[data-submit='progress_bar']").serialize().replace('method=put', 'method=post');
+  var data = $("form").serialize().replace('method=put', 'method=post');
```

Comment:
what's the difference in other places that still uses [data-submit='progress_bar']? does it mean they are broken on hostgroup edit?